dask/dataframe/io/tests/test_csv.py

```diff
@@ -260,6 +275,36 @@ def test_read_csv(dd_read, pd_read, text, sep):
         assert_eq(result, pd_read(fn, sep=sep))
 
 
+@pytest.mark.parametrize('dd_read,pd_read,text,skip',
+                         [(dd.read_csv, pd.read_csv, csv_text, 7),
+                          (dd.read_table, pd.read_table, tsv_text, [1, 13])])
+def test_read_csv_large_skiprows(dd_read, pd_read, text, skip):
+    names = ['name', 'amount']
+    with filetext(text) as fn:
+        actual = dd_read(fn, skiprows=skip, names=names)
+        assert_eq(actual, pd_read(fn, skiprows=skip, names=names))
+
+
+@pytest.mark.parametrize('dd_read,pd_read,text,skip',
+                         [(dd.read_csv, pd.read_csv, csv_text, 7),
+                          (dd.read_table, pd.read_table, tsv_text, [1, 12])])
+def test_read_csv_skiprows_only_in_first_partition(dd_read, pd_read, text, skip):
+    # If sample is larger than the first partition this behavior is expected
+    names = ['name', 'amount']
+    with filetext(text) as fn:
+        with pytest.warns(UserWarning) as w:
+            actual = dd_read(fn, blocksize=200, skiprows=skip, names=names).compute()
+            assert_eq(actual, pd_read(fn, skiprows=skip, names=names))
+            assert len(w) == 1
+            msg = str(w[0].message)
+            assert 'sample=blocksize' in msg
+
+        with pytest.warns(UserWarning) as w:
+            with pytest.raises(ValueError):
```

Comment:
Ah sorry missed this comment. Line 305 was left over from an early test I wrote to show this behavior before I added the setting sample to blocksize part. I'm taking it out now since you are right - it never gets run.